src/main/java/de/dennisguse/opentracks/adapters/IntervalStatisticsAdapter.java

```diff
@@ -0,0 +1,65 @@
+package de.dennisguse.opentracks.adapters;
+
+import android.content.Context;
+import android.util.Pair;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ArrayAdapter;
+import android.widget.TextView;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import java.util.List;
+
+import de.dennisguse.opentracks.R;
+import de.dennisguse.opentracks.util.PreferencesUtils;
+import de.dennisguse.opentracks.util.StringUtils;
+import de.dennisguse.opentracks.viewmodels.IntervalStatistics;
+
+public class IntervalStatisticsAdapter extends ArrayAdapter<IntervalStatistics.Interval> {
+
+    private boolean metricUnits;
+    private float accDistance = 0f;
+
+    public IntervalStatisticsAdapter(Context context, List<IntervalStatistics.Interval> intervalList) {
+        super(context, R.layout.interval_stats_list_item, intervalList);
+        metricUnits = PreferencesUtils.isMetricUnits(context);
+    }
+
+    @NonNull
+    @Override
+    public View getView(int position, @Nullable View intervalView, @NonNull ViewGroup parent) {
+        IntervalStatistics.Interval interval = getItem(position);
+        ViewHolder viewHolder;
+
+        if (intervalView == null) {
+            viewHolder = new ViewHolder();
+
+            intervalView = LayoutInflater.from(getContext()).inflate(R.layout.interval_stats_list_item, parent, false);
+
+            viewHolder.distance = intervalView.findViewById(R.id.interval_item_distance);
+            viewHolder.speed = intervalView.findViewById(R.id.interval_item_speed);
+            viewHolder.pace = intervalView.findViewById(R.id.interval_item_pace);
+
+            intervalView.setTag(viewHolder);
+        } else {
+            viewHolder = (ViewHolder) intervalView.getTag();
+        }
+
+        accDistance += interval.getDistance();
+        viewHolder.distance.setText(StringUtils.formatDistance(getContext(), accDistance, metricUnits));
+        Pair<String, String> speedParts = StringUtils.getSpeedParts(getContext(), interval.getSpeed(), metricUnits, true);
+        viewHolder.speed.setText(speedParts.first + " " + speedParts.second);
+        viewHolder.pace.setText(StringUtils.getSpeedParts(getContext(), interval.getSpeed(), metricUnits, false).first);
```

Comment:
Because pace is time. When you see stats in a table, the column with pace doesn't show unit. I've never seen the unit but It's only my experience and I don't know what standard says about this (if any standard about it).

Reply:
One the statistics fragment pace is shown with unit.
i don't know if it is meaningful to the user.

Reply:
You are right... So I have to add here the unit too for maintain the coherence.

Anyway, I'm going to look into about this matter... For the moment I'll add the unit

;)